docs/conf.py

```diff
@@ -114,8 +114,8 @@
     # "left_sidebar_end": ["custom-template.html", "sidebar-ethical-ads.html"],
     # "footer_items": ["copyright", "sphinx-version", ""]
     "switcher": {
-        # "json_url": "/_static/switcher.json",
-        "json_url": "https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/switcher.json",
+        "json_url": "/_static/switcher.json",
```

Comment:
This is going to break when it is merged.
It is nice to have as is now to quickly check a functional button, but it should be reverted before merging it: https://github.com/pydata/pydata-sphinx-theme/pull/436#issuecomment-953097034
I am currently thinking of alternatives to get rid of this ugly pattern...

Reply:
good catch - switched it back in this PR